notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb

```diff
@@ -0,0 +1,301 @@
+    "## Download Complete\n",
+    "You have now downloaded all the TESS DV products for this star and thier locations can be seen by printing the \"Local Path\" in the manifest."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print( manifest['Local Path'] )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examine the download manifest\n",
+    "\n",
+    "TESS file names tell you a lot about what is in the file. In the function `parse_manifest` below I break them apart so that I can make an easy to read table about the type of data that we downloaded.  Then we write out that part of the table.   This makes it obvious that there are lots of different sets of DV files based on different searches, each with a different sector range.  For each sector it was observed, there is a single sector search, and then there are also several multi sector searches."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def parse_manifest(manifest):\n",
+    "    \"\"\"\n",
+    "    Parse manifest and add back columns that are useful for TESS DV exploration.\n",
+    "    \"\"\"\n",
+    "    results = deepcopy(manifest)\n",
+    "    filenames = []\n",
+    "    sector_range = []\n",
+    "    exts = []\n",
+    "    for i,f in enumerate(manifest['Local Path']):\n",
+    "        file_parts = np.array(np.unique(f.split(sep = '-')))\n",
+    "        sectors = list( map ( lambda x: x[0:2] == 's0', file_parts))\n",
+    "        s1 = file_parts[sectors][0]\n",
+    "        try:\n",
+    "            s2 = file_parts[sectors][1]\n",
+    "        except:\n",
+    "            s2 = s1\n",
+    "        sector_range.append(\"%s-%s\" % (s1,s2))\n",
+    "        path_parts = np.array(f.split(sep = '/'))\n",
+    "        filenames.append(path_parts[-1])\n",
+    "        exts.append(path_parts[-1][-8:])\n",
+    "\n",
+    "    results.add_column(table.Column(name = \"filename\", data = filenames))\n",
+    "    results.add_column(table.Column(name = \"sectors\", data = sector_range))\n",
+    "    results.add_column(table.Column(name = \"fileType\", data = exts))\n",
+    "    results.add_column(table.Column(name = \"index\", data = np.arange(0,len(manifest))))\n",
+    "    \n",
+    "    return results\n",
+    "\n",
+    "#Run parser and print\n",
+    "results = parse_manifest(manifest)\n",
+    "print(results['index','sectors','fileType'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plot the DVT File\n",
+    "The time series data use to find each transit signal (or Threshold Crossing event (TCE)) is found in the dvt.fits files.  As you can see there is a dvt file. If we want the file with the most data, we should pick the one with the longest sector range in the file name.  In this case it is s0001-s0013. "
```

Comment:
use -> used
Suggest rewording the TCE language as:
each transit signal (or Threshold Crossing event (TCE)) -> each Threshold Crossing Event (TCE), the putative transit signal,

Reply:
Not sure why, but I don't like the work putative.  See if what I wrote instead is clear.

Reply:
Heh, I overuse putative so much.  What you wrote is clear.